core/src/codegen/attr_extractor.rs

```diff
@@ -14,6 +15,9 @@ pub trait ExtractAttribute {
 
     fn forwarded_attrs(&self) -> Option<&ForwardAttrs>;
 
+    /// Gets the field that will receive forwarded attributes.
+    fn attrs_field(&self) -> Option<&AttrsField>;
```

Comment:
I think it might make sense to combine `ForwardAttrs` and `AttrsField` into one struct that represents both the forwarding policy and the receiving field.

Reply:
This has now been done.